Web/scripts/fluid_properties.IF97Profile.py

```
@@ -0,0 +1,194 @@
+        s.fast_evaluate(CP.HmassP_INPUTS, h, p, outputs, out, status)
+        samples.append((time.perf_counter_ns() - t0) / h.size)
+    return samples, True
+
+
+def time_manual_batch(s, h: np.ndarray, p: np.ndarray, n_runs: int) -> list[float]:
+    """Time ``[update(...) for each point]`` inside one tick block."""
+    for k in range(h.size):  # warmup
+        try:
+            s.update(CP.HmassP_INPUTS, float(h[k]), float(p[k]))
+        except Exception:
+            pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)